packages/icon/src/template.ts

```diff
@@ -7,6 +7,7 @@ interface BabelAPI extends Record<string, any> {
 interface SVGROptions extends Record<string, any> {
   state: {
     componentName: string;
+    size: 'small' | 'default' | 'large' | 'xlarge' | number;
```

Comment:
I don't think this belongs in `SVGROptions`.

Reply:
👍